apps/meteor/server/models/raw/Sessions.ts

```diff
@@ -991,31 +991,13 @@ export class SessionsRaw extends BaseRaw<ISession> implements ISessionsModel {
 		return [
 			{ key: { createdAt: -1 } },
 			{ key: { loginAt: -1 } },
+			{ key: { searchTerm: 1 }, partialFilterExpression: { searchTerm: { $exists: true } }, background: true },
```

Comment:
not sure what is the impact of `partialFilterExpression` here, since it mention the same field of the index.

Reply:
`searchTerm` can be undefined on old documents. In this case, `partialFilterExpression` just will add to the index if the object exists.